.dotnet/tests/Batch/BatchTests.cs

```diff
@@ -22,14 +23,14 @@ public BatchTests(bool isAsync) : base(isAsync)
     }
 
     [Test]
-    public async Task ListBatchesProtocol()
+    public void ListBatchesProtocol()
     {
         BatchClient client = GetTestClient();
-        ClientResult result = IsAsync
-            ? await client.GetBatchesAsync(after: null, limit: null, options: null)
-            : client.GetBatches(after: null, limit: null, options: null);
+        var enumerator = client.GetBatches(after: null, limit: null, options: null).GetEnumerator();
```

Comment:
It would be preferable, I think, to test these APIs through the public interface, i.e. without the call to GetEnumerator.  We should validate that the end-user usage of the APIs works, so that the helper wrappers get test coverage as well.